src/layers/legacy/medVtkInria/vtkImageView/vtkImageView.cxx

```diff
@@ -1773,6 +1758,8 @@ void vtkImageView::SetCurrentLayer(int layer)
         }
         this->ScalarBar->Modified();
         this->Modified();
+
+        this->Render();
```

Comment:
when you click to an other layer in the layer list, the patient, study and series names will be updated because of this render

Reply:
ok